library/ccm.c

```diff
@@ -403,7 +404,13 @@ static const unsigned char res[NB_TESTS][32] = {
 int mbedtls_ccm_self_test( int verbose )
 {
     mbedtls_ccm_context ctx;
-    unsigned char out[32];
+    /*
+     * Some HW accelerators require the input and output buffers would be in RAM,
+	 * because the falsh is not accessible. Use buffers on the stack, to hold the
+	 * test vectors data.
```

Comment:
- Replace `HW` with `hardware`.
- This wrapping isn't quite right. Please fix.
- Replace `falsh` with `flash`
- Remove comma after `stack`